<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeroList_window</class>
 <widget class="QDialog" name="HeroList_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>3</y>
     <width>641</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_axe">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>120</width>
        <height>100</height>
       </size>
      </property>
      <property name="NICKNAME" stdset="0">
       <string>AXE</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_abaddon">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>120</width>
        <height>100</height>
       </size>
      </property>
      <property name="NICKNAME" stdset="0">
       <string>ABADDON</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_apparat">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>120</width>
        <height>100</height>
       </size>
      </property>
      <property name="NICKNAME" stdset="0">
       <string>APPARAT</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_antimage">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>120</width>
        <height>100</height>
       </size>
      </property>
      <property name="NICKNAME" stdset="0">
       <string>ANTIMAGE</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_alchemist">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>120</width>
        <height>100</height>
       </size>
      </property>
      <property name="NICKNAME" stdset="0">
       <string>ALCHEMIST</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_arc">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>120</width>
        <height>100</height>
       </size>
      </property>
      <property name="NICKNAME" stdset="0">
       <string>ARCWARDEN</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
